epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -94,6 +94,11 @@ String getterCode() {
     return isPrivate ? getterMethodName + "()" : name;
   }
 
+  // Special case to avoid generating recursive getter if field and its getter names are the same
+  String superGetterCode() {
```

Comment:
would it be possible to combine this with the existing `getterCode` method? I did something similar with `setterCode()` to handle the super case.

I would worry about mixing up `superGetterCode` and `getterCode` and confusion over when to use each. It seems better to have one method that handles it all.